package/yast2-installation-control.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Aug 22 15:58:13 CEST 2018 - hellcp@opensuse.org
+
+- Add a way to switch to installation ui (boo#1088785)
```

Comment:
Although approved, probably would modify the wording as it suggests that we are allowing to switch to an installation UI when we are just modifying the theme.